### DISCUSSION TITLE
Build error I can't fix

### DISCUSSION THREAD

**Discussion by Johannesen007** [picocomputer/community#55] [Q&A] [answered]
When I build the Scaffold "Hello world" I get the error:
[build] E:/source/GitHub/rp6502-vscode/rp6502-vscode-main/rp6502-sdk/src/ria.s:88: Error: Unresolved external '__mappederrno'

If I modify the ria.s file so it does not use the __mappederrno it compiles without problems, and loads on the picocomputer.

So I guess everything vscode, CC65, msys2 and stuff is installed correctly and paths is ok, but I don't understand why i get the "Unresolved external" error.

Anyone who no what could be wrong?

**Comment by rumbledethumps** [comment]
The CC65 project has a new maintainer who decided to change all the internals. Use an older version. I'm on V2.18.

**Reply by rumbledethumps** [reply, marked as answer]
https://github.com/cc65/cc65/discussions/2261

**Comment by Johannesen007** [comment]
Thanx a lot, i changed the __mappederrno to ___mappederrno in ria.s, and now it works 👍 . there might be other places, and I will meet then as time goes by. I am also aware that I need to make the changes again if i download an updated version of the sdk at some point.
